runtime/cudaq/platform/default/CMakeLists.txt

```diff
@@ -40,4 +40,6 @@ endif()
 if (CUSTATEVEC_ROOT AND CUDA_FOUND)
   add_target_config(nvidia)
   add_target_config(nvidia-fp64)
-endif()
\ No newline at end of file
+endif()
+
+add_target_config(photonics)
```

Comment:
Can you move this call to `runtime/cudaq/qis/managers/photonics/CMakeLists.txt`, and move the file into that folder as well?

Reply:
Moved `add_target_config(photonics)` runtime/cudaq/qis/managers/photonics/CMakeLists.txt
Moved photonics.config to runtime/cudaq/qis/managers/photonics/